frontend/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>ConectDev`s</title>
+    <link rel="stylesheet" href="./src/styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100..900;1,100..900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <main class="main">
+      <div class="container__image">
+        <img src="./src/assets/image-login.png" alt="" />
+      </div>
+      <div class="container__form" id="login-form">
+        <img src="./src/assets/logo.jpeg" alt="" />
+        <div>
+          <h1>Acesse sua conta</h1>
+          <form id="login-form-action">
+            <input type="email" placeholder="Email" required />
+            <input type="password" placeholder="Senha" required />
+            <button type="submit">Acessar</button>
+          </form>
+        </div>
+        <p>
+          Não tenho conta.
+          <button class="btn__redirect" id="show-register">
+            Criar conta agora
+          </button>
+        </p>
+      </div>
+
+      <div class="container__form" id="register-form">
+        <img src="./src/assets/logo.jpeg" alt="" />
+        <div>
+          <h1>Preencha os campos abaixo</h1>
+          <form id="register-form-action">
+            <input type="text" placeholder="Nome" name="name" required />
+            <input type="email" placeholder="Email" required />
+            <input type="password" placeholder="Senha" required />
+            <input type="password" placeholder="Confirme a senha" required />
+            <button type="submit">Acessar</button>
```

Comment:
```suggestion
            <button type="submit">Criar minha conta</button>
```
Esse botão é o de criar conta